docs/docsite/rst/dev_guide/developing_collections.rst

```diff
@@ -171,8 +171,18 @@ TBD.
 tests directory
 ----------------
 
-TBD. Expect tests for the collection itself to reside here.
+Executing sanity, integration, and unit tests is largely identical to performing these tests within Ansible's mainline code. ``ansible-test`` executes against modules implemented from the relative path ``collections/.. For example, ``ansible-test`` may be executed against IOS modules from the default collection location of ``~/.ansible/collections/cisco/ios/``. Tests may also be executed while in a custom directory, such as ``~/work/ansible/collections/cisco/ios/``, as long as it is the current path.
```

Comment:
The important directory is `ansible_collections` rather than `collections`:

```suggestion
Executing sanity, integration, and unit tests is largely identical to performing these tests within Ansible's mainline code. ``ansible-test`` executes against modules implemented from the relative path ``ansible_collections/..`` For example, ``ansible-test`` may be executed against IOS modules from the default collection location of ``~/.ansible/collections/ansible_collections/cisco/ios/``. Tests may also be executed while in a custom directory, such as ``~/work/ansible_collections/cisco/ios/``, as long as it is the current path.
```

Two other things should also be explained:

1. The `ansible_collections` directory must not be within the Ansible source tree. If it is, `ansible-test` will try to test the Ansible source instead of the collection.
2. Symbolic links cannot be used to construct the required path. The real path is seen by Ansible and other tools.